src/app/SharpRaven/Data/ExceptionFrame.cs

```diff
@@ -219,5 +223,11 @@ public override string ToString()
 
             return sb.ToString();
         }
+
+        private static bool IsSystemModuleName(string moduleName)
+        {
+            return !string.IsNullOrEmpty(moduleName) &&
+                moduleName.StartsWith("System.", System.StringComparison.Ordinal);
```

Comment:
@marcomorain Configuring a `Func<string, bool>` sounds most flexible, yea. I'm ok with merging this without configurability for now.